src/main/java/frc/robot/util/RobotMath.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.util;
+
+public class RobotMath {
+    public static double powerCurve(double x) {

```

Comment:
This does not belong here. As is, it causes confusion with the `powerCurve` method in `TwistArm`. They do similar things, but have entirely different implementations for doing so, and putting one of them in this `public` place causes enough confusion to be setting off red flags for me. 🚩

The appropriate place for this method is a `private static` member of `Drive`.